docs/iris/src/developers_guide/graphics_tests.rst

```diff
@@ -10,9 +10,10 @@ For this, a basic 'graphics test' assertion operation is provided in the method
 match against a stored reference.
 A "graphics test" is any test which employs this.
 
-At present (Iris version 1.10), such tests include the testing for modules
-`iris.tests.test_plot` and `iris.tests.test_quickplot`, and also some other
-'legacy' style tests (as described in :ref:`developer_tests`).
+At present, such tests include the testing for modules `iris.tests.test_plot`
+and `iris.tests.test_quickplot`, all output plots from the gallery examples
+(contained in `docs/iris/example_tests`), and a few  other 'legacy' style tests
+(as described in :ref:`developer_tests`).
```

Comment:
Where in `tests.rst` are the legacy style tests described? I can't find any reference to them there.

Reply:
Right at the top it says "There are three categories of tests within Iris: / * Unit tests / * Integration tests / * Legacy tests"

Reply:
Indeed, however the only subsequent mention in that page is how legacy tests should be avoided. If there are any legacy style tests that use graphics tests they are not described in the page.

Reply:
Actually, what "legacy" means in this case is just "before we defined a separation between integration and unit tests".  That is, all the "iris/tests/test_*.py".
The implication is, we can't be bothered to remove/restructure all that, but we don't want any more like it.

I'm basically just not convinced that trying to spell that out will add anything very worthwhile ...

Reply:
OK. There doesn't seem to be a way out of this that isn't confusing.